### PR TITLE
DDF-3733 Check if both sort keys are undefined

### PR DESCRIPTION
**Pull request by brendan-hofmann**
#### What does this PR do?
Fix sorting when some values are undefined.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@adimka 
@mackncheesiest 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/ui
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@millerw8
#### How should this be tested? (List steps with links to updated documentation)
Build and install. Ingest some metacards and add a new field (e.g. topic.keyword) to some of them. Sort by keyword, then by title. Verify that when no keyword is defined, the metacards are still sorted by title.
#### What are the relevant tickets?
[DDF-3733](https://codice.atlassian.net/browse/DDF-3733)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "version": "0.0.1",
   "license": "LGPL-3.0",
   "scripts": {
-    "startplus": "webpack-dev-server --config=./webpack/config/devServer.js --open --port 8080 --host 0.0.0.0",
+    "startplus": "webpack-dev-server --config=./webpack/config/devServer.js --open --port 8080 --host localhost",
```
[the remark this answers is not in the document]
Comment (brendan-hofmann) [reply ui/packages/catalog-ui-search/package.json]: I can split it if you feel strongly but live-reload is broken on Windows without this change.

**Comment by bdeining** [comment]
build now

**Comment by mackncheesiest** [comment]
Hero looks good on Mac OS X + Firefox

Ingested two records and changed the first sort attribute to `Topic.Keyword` which neither of them had
Added a new sort attribute for `title` and verified that my results are sorted both in the results list and in the table view when I change between `A to Z` and `Z to A` sorting.

**Comment by andrewkfiedler** [comment]
build now

**Pull request merged**